### PR TITLE
Fix yarn dev script

### PR DESCRIPTION
**Pull request by cdce8p**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "clean": "rm -rf build/* build-temp/* build-es5/* build-temp-es5/* build-translations/*",
     "build": "script/build_frontend",
-    "dev": "npm run gulp ru_all gen-service-worker",
```
Comment (balloob) [inline package.json]: This should just run `script/develop` or even better, just be removed.

**Comment by cdce8p** [comment]
I removed it. However `./node_modules/.bin/gulp gen-service-worker` needs to be executed at least once if I'm not mistaken. The frontend didn't work without it for me. Therefore I added it to `script/bootstrap`.

**Comment by cdce8p** [comment]
I also opened a PR for the `dev` docs (for `script/develop`): https://github.com/home-assistant/developers.home-assistant/pull/15

File: script/bootstrap [hunk 2]
```diff
@@ -16,4 +16,4 @@ yarn install
 script/build_frontend
 
 # Build files that need to be generated to run development mode
-yarn dev
+./node_modules/.bin/gulp gen-service-worker
```
Comment (balloob) [inline script/bootstrap]: Bootstrap should just run yarn. All other lines can go in favor of `script/devel`

**Review by balloob — APPROVED** [review]

**Pull request merged**